### DISCUSSION TITLE
ECML type content H5p and youtube link getting error.

### DISCUSSION THREAD

**Discussion by sagarkoshti1990** [Sunbird-Ed/Community#242] [General Discussion]
## Description
We have integrated sunbird content player in our project using Webcomponent.
In that ECML type content  `application/vnd.ekstep.h5p-archive`  and `video/x-youtube` mimeType
content getting error can someone share inputs to resolve this ?

## video/x-youtube

![Screenshot from 2022-11-02 17-05-06](https://user-images.githubusercontent.com/19996753/201026569-8d53da97-9067-455d-9c67-3028771220e5.png)

## application/vnd.ekstep.h5p-archive

![Screenshot from 2022-11-02 17-09-50](https://user-images.githubusercontent.com/19996753/201026743-6556db36-7720-40f1-8ea7-073e136e1479.png)

**Comment by Krishnaj20** [comment]
@sharathkashyap - Pls take care of this.
